### PR TITLE
Seeds for banners and campaigns

### PR DESCRIPTION
**Pull request by c3zi**
```
# ./artisan migrate:fresh && ./artisan db:seed
```

### CODE REVIEW

**Review by teghnet — DISMISSED** [review]

**Review by teghnet — APPROVED** [review]

File: app/Services/Adselect.php [hunk 1]
```diff
@@ -95,21 +93,21 @@ public function getBanners(array $requests)
...
-
-        foreach ($requests as $request) {
-            $responses[] = [
-                'request_id' => $request['request_id'],
-                'banner_id' => $request['request_id'] + 1,
-            ];
-        }
-
-        return $responses;
+//        $responses = [];
```
Comment (adshares-tech) [inline app/Services/Adselect.php:96]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 2]
```diff
@@ -33,11 +31,11 @@ class Adselect
     public function __construct($endpointUrl, $debug = false)
     {
         $this->endpointUrl = $endpointUrl;
-        $this->rpcClient = new Client($this->endpointUrl);
-        $this->rpcClient->getHttpClient()->withTimeout(1);
-        if ($debug) {
-            $this->rpcClient->getHttpClient()->withDebug();
-        }
+//        $this->rpcClient = new Client($this->endpointUrl);
```
Comment (adshares-tech) [inline app/Services/Adselect.php:34]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 3]
```diff
@@ -67,7 +65,7 @@ public function addImpressions(array $events)
 
     public function getBanners(array $requests)
```
Comment (adshares-tech) [inline app/Services/Adselect.php:66]: MAJOR Remove the unused function parameter "$requests". [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)

File: app/Services/Adselect.php [hunk 4]
```diff
@@ -95,21 +93,21 @@ public function getBanners(array $requests)
...
-            $responses[] = [
-                'request_id' => $request['request_id'],
-                'banner_id' => $request['request_id'] + 1,
-            ];
-        }
-
-        return $responses;
+//        $responses = [];
+//
+//        foreach ($requests as $request) {
```
Comment (adshares-tech) [inline app/Services/Adselect.php:98]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 5]
```diff
@@ -67,7 +65,7 @@ public function addImpressions(array $events)
 
     public function getBanners(array $requests)
     {
-        return $this->rpcClient->execute('banner_select', $requests);
+//        return $this->rpcClient->execute('banner_select', $requests);
```
Comment (adshares-tech) [inline app/Services/Adselect.php:68]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 6]
```diff
@@ -95,21 +93,21 @@ public function getBanners(array $requests)
...
-        }
-
-        return $responses;
+//        $responses = [];
+//
+//        foreach ($requests as $request) {
+//            $responses[] = [
+//                'request_id' => $request['request_id'],
+//                'banner_id' => $request['request_id'] + 1,
+//            ];
```
Comment (adshares-tech) [inline app/Services/Adselect.php:102]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 7]
```diff
@@ -95,21 +93,21 @@ public function getBanners(array $requests)
...
+//        }
+//
+//        return $responses;
```
Comment (adshares-tech) [inline app/Services/Adselect.php:105]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 8]
```diff
@@ -138,7 +136,7 @@ public function addCampaigns(array $campaings)
 
     public function deleteCampaigns(array $campaignIds)
```
Comment (adshares-tech) [inline app/Services/Adselect.php:137]: MAJOR Remove the unused function parameter "$campaignIds". [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)

File: app/Services/Adselect.php [hunk 9]
```diff
@@ -138,7 +136,7 @@ public function addCampaigns(array $campaings)
...
     {
-        return $this->rpcClient->execute('campaign_delete', $campaignIds);
+//        return $this->rpcClient->execute('campaign_delete', $campaignIds);
```
Comment (adshares-tech) [inline app/Services/Adselect.php:139]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 10]
```diff
@@ -95,21 +93,21 @@ public function getBanners(array $requests)
...
+//            $responses[] = [
+//                'request_id' => $request['request_id'],
+//                'banner_id' => $request['request_id'] + 1,
+//            ];
+//        }
+//
+//        return $responses;
     }
 
     public function addCampaigns(array $campaings)
```
Comment (adshares-tech) [inline app/Services/Adselect.php:108]: MAJOR Remove the unused function parameter "$campaings". [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)

File: app/Services/Adselect.php [hunk 11]
```diff
@@ -95,21 +93,21 @@ public function getBanners(array $requests)
...
     {
-        return $this->rpcClient->execute('campaign_update', $campaings);
+//        return $this->rpcClient->execute('campaign_update', $campaings);
```
Comment (adshares-tech) [inline app/Services/Adselect.php:110]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 12]
```diff
@@ -51,7 +49,7 @@ public function getStatus()
 
     public function addImpressions(array $events)
```
Comment (adshares-tech) [inline app/Services/Adselect.php:50]: MAJOR Remove the unused function parameter "$events". [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)

File: app/Services/Adselect.php [hunk 13]
```diff
@@ -51,7 +49,7 @@ public function getStatus()
...
     {
-        return $this->rpcClient->execute('impression_add', $events);
+//        return $this->rpcClient->execute('impression_add', $events);
```
Comment (adshares-tech) [inline app/Services/Adselect.php:52]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Services/Adselect.php [hunk 14]
```diff
@@ -33,11 +31,11 @@ class Adselect
     public function __construct($endpointUrl, $debug = false)
```
Comment (adshares-tech) [inline app/Services/Adselect.php:31]: MAJOR Remove the unused function parameter "$debug". [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)

File: app/Http/Controllers/SupplyController.php [hunk 15]
```diff
@@ -215,13 +215,16 @@ public function logNetworkView(Request $request, Adselect $adselect, $id)
...
-        $aduser_endpoint = config('app.aduser_endpoint');
+//        $aduser_endpoint = config('app.aduser_endpoint');
+        $aduser_endpoint = config('app.aduser_local_endpoint');
 
         if (empty($aduser_endpoint) || empty($impressionId)) {
             $log->save();
         // TODO: process?
         } else {
-            $userdata = json_decode(file_get_contents("{$aduser_endpoint}/getData/{$impressionId}"), true);
+//            $userdata = json_decode(file_get_contents("{$aduser_endpoint}/getData/{$impressionId}"), true);
```
Comment (adshares-tech) [inline app/Http/Controllers/SupplyController.php:225]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

File: app/Http/Controllers/SupplyController.php [hunk 16]
```diff
@@ -215,13 +215,16 @@ public function logNetworkView(Request $request, Adselect $adselect, $id)
 
         // GET kewords from aduser
         $impressionId = $request->query->get('iid');
-        $aduser_endpoint = config('app.aduser_endpoint');
+//        $aduser_endpoint = config('app.aduser_endpoint');
```
Comment (adshares-tech) [inline app/Http/Controllers/SupplyController.php:218]: MAJOR Remove this commented out code. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)

**Comment by adshares-tech** [comment]
SonarQube analysis reported 17 issues
* ![MAJOR][MAJOR] 17 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [Adselect.php#L29](https://github.com/adshares/adserver/blob/a8e7f3d25bc9adfa35d0a6c531bdf11e87a0d67b/app/Services/Adselect.php#L29): Remove this unused "$rpcClient" private field. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1068)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by adshares-tech** [comment]
SonarQube analysis reported 17 issues
* ![MAJOR][MAJOR] 17 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [Adselect.php#L29](https://github.com/adshares/adserver/blob/e30f75b8e51e6794af89d3e25e728da16008fc27/app/Services/Adselect.php#L29): Remove this unused "$rpcClient" private field. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1068)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by adshares-tech** [comment]
SonarQube analysis reported 17 issues
* ![MAJOR][MAJOR] 17 major

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [Adselect.php#L29](https://github.com/adshares/adserver/blob/33caabb8a301746ffcf76aad112ea7b5350df940/app/Services/Adselect.php#L29): Remove this unused "$rpcClient" private field. [rule](https://sonarcloud.io/coding_rules#rule_key=php%3AS1068)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**